File: loop.go
package main
import "fmt"
import "time"
import "runtime"

func main() {
  start := time.Now()
  var sum, i int64 = 0, 0
  for i = 0; i< 10000000; i++ {
    sum += i
  }
  end := time.Now()
  elapsed := (end.Sub(start)).Seconds()
  fmt.Printf("Go:\t\t%.6f\t%v\t%s\n", elapsed, sum, runtime.Version())
}

File: loop.rs
use std::time::Instant;

fn main() {
  let now = Instant::now();
  let mut sum = 0u64;
  for i in 0..10000000 {
    sum += i
  }
  println!("Rust:\t\t{}.{}\t{}", now.elapsed().as_secs(), now.elapsed().subsec_nanos(), sum)
}
